components/ws-manager-bridge/src/cluster-service-server.ts

```diff
@@ -235,6 +235,46 @@ export class ClusterService implements IClusterServiceServer {
                         }
                     }
                 }
+                if (call.request.hasTls()) {
+                    const tls = req.tls;
+                    if (!tls?.ca || !tls?.crt || !tls?.key) {
+                        throw new GRPCError(grpc.status.INVALID_ARGUMENT, "missing required TLS config");
+                    }
+                    if (tls.ca === cluster.tls?.ca && tls.crt === cluster.tls?.crt && tls.key === cluster.tls?.key) {
+                        callback(null, new UpdateResponse());
+                        return;
+                    }
+
+                    const newCluster: WorkspaceCluster = {
+                        name: req.name,
+                        url: cluster.url,
+                        region: cluster.region,
+                        state: cluster.state,
+                        score: cluster.score,
+                        maxScore: 100,
```

Comment:
Why this magic value? :eyes:

Reply:
Workspace opened now, I see! :+1:

Reply:
@iQQBot You can test by adding the same cluster again, with a different name. Then alter the TLS config manually in the DB. And then update it again (with the original TLS config). It should override.

Starting a workspace does not need to be part of the test (but also can be, if you remove the WS_MANAGER config env var from `server`).